app/src/test/java/uk/ac/ebi/atlas/search/ToBeDeprecatedJsonGeneSearchControllerWIT.java

```diff
@@ -207,156 +210,37 @@ void jsonPayloadContainsFacetDescription() throws Exception {
                 .andExpect(jsonPath("$.checkboxFacetGroups", contains("Marker genes", "Species")));
     }
 
-    @Test
```

Comment:
You deleted many tests here, why is that? Did you make new tests regarding those ones somewhere?

Reply:
Yes, I deleted those tests as they were duplicated.
Please see my 1st commit in this branch: [Remove all the duplicated tests that also presents in JsonGeneSearchControllerWIT](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/491/commits/8359b4bbf7e4e2f6b93267663eadaa8bf6648d9e)